finalPrep/README.md

```diff
@@ -45,30 +45,36 @@ In Mod 0 you've learned about different techniques for managing your time at Tur
 When you are finished, add screenshots of your calendar so we can provide feedback if needed!
 
 - `Add Week 1 Screenshot Here`
+<img width="1200" alt="Screen Shot 2022-02-10 at 7 03 21 PM" src="https://user-images.githubusercontent.com/93043035/153527462-547f3c3d-470a-49d6-a49c-ad607f51b20c.png">
+
 - `Add Week 2 Screenshot Here`
+
+<img width="1200" alt="Screen Shot 2022-02-10 at 7 03 36 PM" src="https://user-images.githubusercontent.com/93043035/153527520-2c96657d-9100-4ce7-9bb0-2e8e6491b3b9.png">
+
 - `Add Week 3 Screenshot Here`
+<img width="1200" alt="Screen Shot 2022-02-10 at 7 03 50 PM" src="https://user-images.githubusercontent.com/93043035/153527607-0849b078-8c98-4ba1-a536-fad44a093ef4.png">
 
 ## Mentorship Prep
 
 Mentorship is an integral part of the Turing experience and will help jumpstart your technical career. In order to get your mentor relationship started on the right foot, please complete the following deliverables:
 
-- [ ] [Create a profile on Turing Community Connect by following these directions](https://docs.google.com/document/d/1vpyKGu92l1HGkJzULNcyyE72946f4QO1DhQgIz3v1E0/edit?usp=sharing)
-- [ ] Complete the [Mentorship DTR Prep](https://gist.github.com/ericweissman/51965bdcbf42970d43d817818bfaef3c)
-  - [ ] Add link to your gist here: 
+- [X] [Create a profile on Turing Community Connect by following these directions](https://docs.google.com/document/d/1vpyKGu92l1HGkJzULNcyyE72946f4QO1DhQgIz3v1E0/edit?usp=sharing)
+- [x] Complete the [Mentorship DTR Prep](https://gist.github.com/ericweissman/51965bdcbf42970d43d817818bfaef3c)
+  - [x] Add link to your gist here: [See my Mentorship DTR gist here!](https://gist.github.com/ASands17/0a91cf89984d0bc95a1bcd7f6876eb6f)
```

Comment:
This is great, you are going to get so much out of your mentor relationship! I love the ask for advice on balance with family - that is huge. Have you joined the #parenturing channel on Slack? Might be a good place to connect with other parents.